packages/dashboard/src/app/views/exploreTemplates/index.js

```diff
@@ -136,6 +142,44 @@ function ExploreTemplates() {
     [handleCreateStoryFromTemplate]
   );
 
+  const handleDetailsToggle = useCallback(
```

Comment:
The idea behind `templateActions` and `storyActions` was to allow us to pass in an arbitrary number of actions through components without having to update propTypes or values getting passed through every time an action changed. The dashboard's grid is getting increasingly flat, so I do get what ya did. 

What do you think about adding `handleDetailsToggle` to `templateActions` above this on line 138 - I think this would save you a bunch of prop updates and simplify the structure.  If you go this route, there's a `TemplateActionsPropType` that will also need a quick little update.

Reply:
Yes. Great idea.